config/initializers/version.rb

```diff
@@ -0,0 +1,32 @@
+require 'mkmf'
+
+module Version
+
+  # Version module
+  # Makes the app version available to the application itself
+  # Needs the git executable for all git operations
+
+  git = find_executable('git')
+  VERSION = File.read(File.join(Rails.root,'VERSION')).chomp rescue nil
```

Comment:
I don't like that `rescue` honestly. I'd prefer that you moved this logic in the `value` method. So instead of:

```ruby
elsif VERSION.present?
  "#{VERSION}"
end
```

It should be:

```ruby
else
  version = Rails.root.join("VERSION")
  File.read(version).chomp if File.exists?(version)
end
```

It's both safer and cleaner.